bundles/binding/org.openhab.binding.plugwise/src/main/java/org/openhab/binding/plugwise/internal/Stick.java

```diff
@@ -77,8 +77,12 @@
     /** Plugwise protocol header code (hex) */
     private final static String PROTOCOL_HEADER = "\u0005\u0005\u0003\u0003";
 
+    /** Carriage return */
+    private final static char CR = '\r';
+    /** Line feed */
+    private final static char LF = '\n';
     /** Plugwise protocol trailer code (hex) */
-    private final static String PROTOCOL_TRAILER = "\r\n";
+    private final static String PROTOCOL_TRAILER = new String(new char[] { CR, LF });
```

Comment:
I know you didn't introduce this, and it's very minor, but there is a recommended order of modifiers in Java which improves readability.  [Here](http://cr.openjdk.java.net/~alundblad/styleguide/index-v6.html#toc-modifiers) is one place that lists the common order:

1. Access modifier (public / private / protected)
1. abstract
1. static
1. final
1. transient
1. volatile
1. default
1. synchronized
1. native
1. strictfp